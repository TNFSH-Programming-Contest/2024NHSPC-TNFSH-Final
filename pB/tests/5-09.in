116 10
1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
61 186156 13337447 316 3485594 14 18 7514566 1080191 331947
0 
3 9 2 7
76
0 
8 2 10 3 6 1 5 4 9
9
0 
5 6 5 10 1 9
78
1 19 5
0 
25
9 13 7 57 3 55 6 102 9 47 9 79 5 69 3 81 2 98 2
9 6 2 7 10 8 9 5 4 3
1
6 53 1 38 8 70 2 31 7 60 1 12 8
3 9 5 1
46
6 4 10 63 3 28 6 70 1 25 8 102 6
9 2 9 7 8 5 1 6 10 4
58
8 78 3 64 7 42 3 100 5 35 7 22 5 91 2 86 4
0 
58
4 29 1 115 6 91 8 82 7
2 4 6
9
0 
8 2 1 4 3 7 10 9 5
56
0 
7 9 10 2 4 1 5 8
14
7 67 5 81 1 54 2 11 6 24 1 33 4 61 7
2 4 5
1
1 52 3
0 
60
0 
8 10 9 5 3 2 1 4 6
30
6 41 10 102 1 70 6 85 2 38 5 30 9
4 7 8 10 2
44
7 50 1 48 8 95 1 97 7 25 10 85 10 67 4
0 
80
1 69 7
2 6 2
15
9 100 9 116 3 52 2 73 8 68 5 95 3 105 9 67 8 16 3
5 8 2 3 9 4
56
1 116 4
6 3 5 10 9 1 8
42
10 44 5 88 2 37 7 17 6 15 10 73 2 43 10 72 6 27 6 75 1
4 9 4 2 7
17
0 
8 1 2 6 4 8 10 5 7
99
4 51 7 58 1 40 2 93 9
0 
1
7 76 9 22 2 96 6 32 9 64 7 57 5 91 9
1 7
10
0 
7 8 7 3 4 9 10 5
53
8 28 6 23 2 101 9 8 4 105 4 82 2 7 5 51 2
4 6 9 2 1
40
4 11 7 51 9 68 2 75 8
3 7 5 4
95
6 106 8 1 7 91 3 105 4 37 9 97 2
6 5 6 7 2 3 1
69
0 
9 2 3 8 1 5 6 7 4 9
28
6 85 4 36 6 106 4 67 1 62 8 81 10
10 10 5 7 8 4 3 2 9 1 6
70
3 100 10 73 3 25 8
8 3 10 5 1 2 9 6 4
37
0 
6 3 8 9 1 5 4
87
7 57 6 34 6 81 5 83 7 98 2 73 5 17 8
2 2 7
30
9 114 10 5 4 95 3 66 4 44 7 45 1 76 10 112 5 98 6
4 3 6 2 5
10
1 24 1
0 
58
10 55 4 58 10 87 1 115 7 104 6 37 8 7 8 42 10 60 9 94 1
6 2 7 6 9 10 8
68
5 50 9 21 10 41 1 91 1 15 3
5 4 3 6 7 10
46
8 27 9 110 1 29 2 55 4 97 9 113 7 16 6 77 6
2 3 6
7
9 31 4 13 9 19 9 85 2 100 1 1 4 6 5 61 3 95 3
6 3 9 4 5 7 6
89
4 13 4 21 6 112 4 29 4
4 4 5 6 8
35
4 84 3 47 1 24 7 74 1
2 6 8
78
0 
10 6 9 3 8 1 5 7 10 2 4
45
5 53 7 106 3 48 4 67 2 108 4
7 2 5 4 9 1 10 8
84
6 49 7 22 6 73 3 71 1 78 6 76 10
2 2 3
23
7 31 9 85 9 103 7 7 2 10 6 92 10 97 4
3 2 5 7
7
9 112 2 13 1 51 10 19 10 55 4 116 7 25 6 92 5 101 10
8 1 2 5 10 4 7 3 8
55
1 10 6
8 4 10 3 7 8 6 2 5
14
2 35 10 10 5
1 4
50
3 47 10 70 5 97 8
1 8
22
7 90 7 113 9 100 9 107 6 92 9 54 5 43 9
5 1 5 7 10 9
11
0 
6 6 5 4 9 3 8
26
0 
7 5 6 1 2 9 10 8
84
6 112 5 106 6 89 7 64 3 47 8 34 10
2 6 3
100
2 28 9 84 7
4 8 7 9 6
55
5 54 6 98 7 20 9 12 3 69 4
8 8 1 9 10 6 5 4 3
40
0 
5 4 6 8 2 10
25
4 112 10 8 5 44 1 116 8
9 7 4 2 6 10 8 1 9 3
46
4 7 6 39 6 13 7 107 7
9 6 2 1 4 10 8 7 5 3
52
2 57 8 102 10
4 4 7 8 9
27
0 
7 6 4 5 8 2 9 3
100
0 
8 6 5 4 10 1 8 7 9
29
0 
6 4 5 3 10 1 7
82
8 36 8 4 10 49 9 16 1 104 7 83 7 67 1 25 1
6 1 5 3 10 4 9
68
1 72 3
0 
12
5 83 10 116 1 17 6 9 7 70 10
5 1 3 2 9 6
69
1 114 4
4 8 7 9 3
9
3 82 5 49 8 100 8
9 10 6 1 4 5 2 7 9 8
17
10 54 1 48 2 62 10 14 2 16 4 46 3 94 3 73 5 86 10 22 1
0 
10
2 2 6 82 10
1 3
26
3 48 6 107 9 71 5
1 7
78
6 80 1 42 10 89 2 59 9 39 7 101 1
3 2 10 1
45
0 
5 6 2 8 9 1
11
1 92 5
5 3 6 2 7 5
97
1 77 6
10 6 8 5 7 10 9 4 3 2 1
12
2 59 9 53 4
6 1 4 2 7 5 3
68
8 39 3 47 8 83 8 116 3 42 1 92 2 37 8 100 5
2 5 9
21
3 79 6 111 9 30 5
10 3 6 8 9 1 4 2 7 10 5
99
9 7 4 99 10 41 2 69 3 22 7 60 8 94 7 61 3 68 2
10 5 9 10 2 4 7 1 3 8 6
3
10 103 7 66 10 37 2 81 9 48 4 98 1 4 9 65 2 1 5 7 8
7 10 9 7 6 5 3 2
20
5 23 3 62 7 6 2 66 7 75 10
10 9 1 7 8 10 4 3 6 2 5
29
6 108 8 77 2 23 4 101 8 57 9 60 9
6 3 8 2 10 7 1
93
0 
10 6 1 8 2 5 3 9 7 10 4
97
9 53 2 98 2 28 9 79 5 10 10 46 6 92 8 14 7 17 9
5 8 5 6 3 1
39
2 91 10 114 10
0 
99
1 21 10
4 4 10 5 3
13
4 109 9 64 4 18 2 50 2
0 
86
5 53 6 14 10 95 2 4 5 91 8
4 1 7 4 10
47
8 32 10 88 2 102 10 20 9 35 6 9 5 100 4 83 1
9 8 1 4 2 10 5 9 3 7
86
10 64 2 57 7 104 7 61 9 74 6 75 8 14 2 71 8 93 1 26 7
5 6 1 5 9 3
39
10 26 7 33 9 53 1 107 5 39 5 94 9 105 7 65 4 88 2 72 3
1 3
90
6 42 7 80 4 53 10 60 10 41 7 103 5
3 8 10 6
6
0 
0 
42
1 39 2
1 4
89
10 83 5 101 1 108 10 73 8 36 1 1 7 19 8 102 4 106 10 27 1
1 6
100
7 82 7 63 10 56 9 80 1 5 4 94 6 65 1
4 5 10 8 1
9
6 20 2 16 3 93 7 43 5 76 8 11 6
1 2
73
10 114 8 83 10 103 7 1 3 38 1 98 10 102 9 82 1 33 5 109 4
9 1 6 3 5 4 2 10 7 9
15
2 77 2 63 3
0 
96
2 75 4 93 1
7 9 1 2 4 6 5 7
44
7 84 2 20 10 19 8 63 6 82 7 72 8 38 1
4 2 9 6 10
96
0 
6 5 2 4 1 6 7
45
0 
1 4
59
1 116 10
9 1 6 7 5 9 4 10 8 2
83
6 96 3 85 2 77 1 74 9 34 8 59 5
8 7 10 4 1 9 3 8 5
58
4 46 7 115 1 108 1 3 2
8 6 9 3 8 1 5 10 7
94
0 
5 1 2 10 9 3
79
7 113 8 32 1 100 7 106 9 73 2 21 5 76 7
9 8 4 6 10 2 7 5 1 3
53
1 110 8
0 
72
9 27 6 26 6 64 1 59 5 1 5 20 3 38 2 103 3 30 5
3 4 8 10
26
3 71 10 56 1 101 1
1 1
3
8 69 7 79 7 102 1 112 2 81 5 62 3 10 8 17 4
9 2 7 1 10 4 8 3 5 6
96
0 
0 
39
4 47 9 116 5 9 2 94 9
5 9 4 5 8 1
43
5 31 7 67 8 64 5 3 10 91 3
2 1 8
75
9 74 9 61 10 1 5 36 4 81 1 49 10 72 4 91 2 45 2
8 10 5 7 8 3 4 9 1
75
0 
8 9 10 6 3 2 8 4 7
18
6 13 9 60 8 2 10 99 4 43 8 95 3
8 6 8 7 3 9 1 2 4
0
