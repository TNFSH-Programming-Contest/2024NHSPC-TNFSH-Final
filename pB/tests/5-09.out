1266441
18421735
4897885
-1
-1
-1
-1
-1
-1
18421786
12598845
-1
-1
18421756
-1
-1
-1
-1
-1
-1
11518771
-1
-1
25750132
-1
-1
-1
25604391
-1
-1
25418262
-1
-1
25750190
-1
-1
-1
-1
-1
-1
25936355
-1
-1
-1
-1
25936660
-1
-1
-1
25418154
12598613
-1
25943638
-1
8033024
-1
-1
-1
25604384
12412736
17155465
-1
-1
-1
-1
-1
-1
-1
-1
-1
8780999
-1
-1
25966590
-1
-1
-1
-1
-1
-1
25936407
-1
-1
24857122
-1
-1
-1
-1
-1
-1
42
-1
-1
-1
-1
-1
-1
-1
-1
3672204
375
-1
-1
-1
14935881
-1
-1
-1
-1
-1
39
-1
-1
-1
22450673
-1
